src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -125,6 +132,10 @@ private void fillInnerParts() {
 
         CommandBox commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
+
+        File iconFile = new File(iconPath);
+        Image iconImage = new Image(iconFile.toURI().toString());
```

Comment:
Would it be better to make this `Image iconImage = new Image(iconPath);` instead? This should work with my above requested change.

Reply:
File object removed